_projects/2024/100535296/100535296.Rmd

```diff
@@ -0,0 +1,534 @@
+---
+title: "Age Distribution of Athletes by Sport since 1896"
+description: |
+  This project focuses on recreating a The Washington Post graph showing the olympic athlete distribution by sport and age since the inception of the modern games in 1896. 
+categories: "2024"
+author: "Irene Bosque Gala"
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+# ORIGINAL GRAPH
+
+The graph titled "Olympians" was published in 2012 by The Washington Post as part of an interactive feature exploring the history of Summer Olympic athletes since the first modern Games in 1896. The visualization aims to break down athlete participation by sport and age group, offering a comprehensive view of how athlete demographics vary across different disciplines, highlighting patterns and outliers in Olympic history.
+
+The visualization organizes sports along the vertical axis, ranging from Archery to Wrestling, while the horizontal axis represents age groups, starting from 10–15 years and increasing gradually. The color gradient is used to indicate the density of athletes within each age group for a specific sport: darker blue shades represent higher participation, while lighter shades indicate smaller numbers. 
+
+
+```{r, fig.width=4, fig.height=6, out.width="100%"}
+#| echo: false
+knitr::include_graphics("C:/Users/irene/OneDrive/Escritorio/MUCSS/Data Visualisation/dataviz/432ccc9bae023c97f2f7a035df4d3e13.jpg")
```

Comment:
Include this file in the PR close to this Rmd, then call the name of the file, without any path.

Also, set `fig.cap="some caption here"` and `out.extra="class=external"` as chunk options.